infra/client/product-planner-ng/src/app/categories/edit-categories/edit-categories.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EditCategoriesComponent } from './edit-categories.component';
+
+describe('EditCategoriesComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/maciejmalecki/inventory/pullRequest?prid=7113829)
```suggestion
describe("EditCategoriesComponent", () => {
```